fastlane/lib/fastlane/actions/crashlytics.rb

```diff
@@ -64,7 +66,6 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :crashlytics_path,
                                        env_name: "CRASHLYTICS_FRAMEWORK_PATH",
                                        description: "Path to the submit binary in the Crashlytics bundle (iOS) or `crashlytics-devtools.jar` file (Android)",
-                                       default_value: Dir["./Pods/iOS/Crashlytics/Crashlytics.framework"].last || Dir["./**/Crashlytics.framework"].last,
```

Comment:
Why did you move this over to `crashlytics_helper.rb`? Isn't this basically the same thing then?